packages/cli/rules.ts

```diff
@@ -36,6 +36,9 @@ export const rulesMessages = {
   [RulesMessage.ERROR_DRIZZLE_R_SERVER]: error(
     `A ${inverse(bold("Server"))} is required when using ${inverse(bold("Drizzle"))}`,
   ),
+  [RulesMessage.ERROR_SENTRY_R_ENV]: error(
+    `A supported environment ${inverse(bold("(React, Vue, Solid, plain Javascript)"))} is required when using ${inverse(bold("Sentry"))}`,
+  ),
```

Comment:
Seems unnecessary, as this is always true. You can just remove the rule entirely.